pkg/rotator/rotator.go

```diff
@@ -68,17 +71,16 @@ func AddRotator(mgr manager.Manager, cr *CertRotator) error {
 		return err
 	}
 
-	vwhKey := types.NamespacedName{Name: cr.VWHName}
 	reconciler := &ReconcileWH{
 		client:        mgr.GetClient(),
 		scheme:        mgr.GetScheme(),
 		ctx:           context.Background(),
 		secretKey:     cr.SecretKey,
-		vwhKey:        vwhKey,
 		crdNames:      cr.CRDNames,
 		wasCAInjected: cr.wasCAInjected,
+		webhooks:      cr.Webhooks,
 	}
-	if err := addController(mgr, reconciler); err != nil {
+	if err := addController(mgr, reconciler, cr.Webhooks); err != nil {
```

Comment:
Why pass in `cr.Webhooks` separately when it's available as reconciler.webhooks?

Reply:
Done